companies/fidelity-de.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
This page (https://www.fidelity.de/ueber-uns/kontakt/unsere-ansprechpartner/) lists a phone and fax number that we should include.